Added activities section

STEdirisuriya opened the pull request:
- [x] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of the change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- https://stedirisuriya.github.io/gci18.fossasia.org/
- Fixes #363 

## Description / Changes
- Added activities section with GitHub API.

## Screenshots if any:
![123](https://user-images.githubusercontent.com/33794760/48853938-a150a000-edd6-11e8-993a-e00693457257.png)

Ryan10145 commented:
@STEdirisuriya Should the Thank You for Contributing section be moved below the new activities section? I feel like the Thank You for Contributing section fits into the footer and looks out of place now that is in the middle of the website.

STEdirisuriya commented:
@Ryan10145 I have added it before the thank you section,
@fragm3 I have removed unwanted file changes. Could you please check it again, if you're free?
and also could you please tag this issue as an advance, https://github.com/fossasia/gci18.fossasia.org/issues/363

Thank you!

Ryan10145 commented on css/custom.css:
```diff
@@ -438,3 +438,44 @@ p.person-name {
+}
+.issues-wrapper i {
+  margin-right: 5px;
+}
```
Add a new line at the end of the file

Ryan10145 commented on js/scripts.js:
```diff
@@ -1437,3 +1437,46 @@ $(document).ready(function(){
+            $(".issues-wrapper").append(html);
+        });
+    });
+});
```
Add a new line at the end of the file

STEdirisuriya commented:
@Ryan10145 I have resolved these issues, thank you for pointing me them.

STEdirisuriya commented:
@Ryan10145 @fragm3 @nikit19 @jogendra Thank you all for reviewing my PR!
